mopidy_spotify_web/library.py

```diff
@@ -97,6 +100,28 @@ def browse(self, uri):
         elif uri == 'spotifyweb:albums':
             return self._cache.sortedAlbums
             # return Ref directory for all albums
+        elif uri.startswith('spotifyweb:featured-playlists') or \
```

Comment:
`uri.startswith(('spotifyweb:featured-playlists', 'spotifyweb:new-releases', 'spotifyweb:categories'))` works btw :-)

Personally I also like to replace this with something like https://github.com/mopidy/mopidy-dirble/blob/master/mopidy_dirble/translator.py#L14-18 so you just the the variant and identifier back to work with.